baybe/utils/plotting.py

```diff
@@ -0,0 +1,78 @@
+"""Plotting utilities."""
+
+import json
+import os
+import sys
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, path: Path, base_name: str, **kwargs) -> None:
+    """Create plots from a given data frame and save them as a svg file.
+
+    The plots will be saved in the location specified by ``path``.
+    The attribute ``base_name`` is used to define the name of the outputs.
+
+    If the ``SMOKE_TEST`` variable is set, no plots are being created and this method
+    immediately returns.
+
+    Using the ``BAYBE_MULTIVERSION_PLOTS`` environment variable, it is possible to
+    create plots for the light and dark version of the documentation. If this variable
+    is not set, a single file named ``{base_name}_check.svg`` is created.
+
+    Args:
+        data: The data frame containing the data to be plotted.
+        path: The path to the directory in which the plots should be saved.
+        base_name: The base name that is used for naming the output files.
+        **kwargs: Keyword arguments. They are directly passed to ``sns.lineplot`` and
+            are used for specifying the plot.
+    """
+    # If we run a SMOKE_TEST, we immediately return
+    if "SMOKE_TEST" in os.environ:
+        return
+    # We see if we happen to find the plotting themes in the current folder.
+    # This is e.g. the case if we convert the file to a jupyter notebook.
+    try:
+        themes = json.load(open("plotting_themes.json"))
+    except FileNotFoundError:
+        # Try to find the plotting themes by backtracking
+        # Get the absolute path of the current script
+        script_path = Path(sys.argv[0]).resolve()
+        # Backtrack until either the "baybe" folder is found or backtracking is no
+        # longer possible
+        while not script_path.name == "baybe" and script_path != script_path.parent:
```

Comment:
for understanding: `script_path == script_path.parent` is only true for the absolute top level root folder right?

Reply:
As far as I know, yes.

Reply:
See here: 
![image](https://github.com/emdgroup/baybe/assets/84016733/4bc444ff-2327-412c-9814-f07edcdb1c34)
Please resolve if this answers your question :)

Reply:
ok i see the `resolve` from above, without it it would prob be dangerous